test/test_jit_cuda_fuser.py

```diff
@@ -657,8 +657,8 @@ def bit_not(x: torch.Tensor):
 
         jitted = torch.jit.script(bit_not)
         x = torch.randn(4, 8, 32, 32, dtype=torch.float, device="cuda").mul(5).to(torch.long)
-        jit_o = bit_not(x)
-        jit_o = bit_not(x)
+        jit_o = jitted(x)
```

Comment:
:face_with_spiral_eyes:

Reply:
Thanks for reviewing. 

The weird thing here is `jitted.graph_for(x)` didn't throw until I tried ld.gold linker. I think it is supposed to be hitting this assertion if we try to get the graph without running it. 

[https://github.com/csarofeen/pytorch/blob/devel/torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp#L678](https://github.com/csarofeen/pytorch/blob/devel/torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp#L678
)
Is that true?